packages/mux-player/src/index.ts

```diff
@@ -674,6 +680,55 @@ class MuxPlayerElement extends VideoApiElement {
     this.setAttribute(PlayerAttributes.SECONDARY_COLOR, `${val}`);
   }
 
+  get defaultShowRemainingTime() {
+    return this.hasAttribute(PlayerAttributes.DEFAULT_SHOW_REMAINING_TIME);
+  }
+
+  set defaultShowRemainingTime(val: boolean | undefined) {
+    if (!val) {
+      this.removeAttribute(PlayerAttributes.DEFAULT_SHOW_REMAINING_TIME);
+    } else {
+      this.setAttribute(PlayerAttributes.DEFAULT_SHOW_REMAINING_TIME, '');
+    }
+  }
+
+  get hideDuration() {
+    return this.hasAttribute(PlayerAttributes.HIDE_DURATION);
+  }
+
+  set hideDuration(val: boolean | undefined) {
```

Comment:
wouldn't this be better suited as a controlsList token? `noduration` 
I'd prefer this to keep the API surface as compact as possible 

this also reminded me of these web component guidelines
https://www.w3.org/2001/tag/doc/webcomponents-design-guidelines/#native-html-elements

> Don’t use custom attributes for styling, like bgcolor. Instead use [Custom Properties](https://drafts.csswg.org/css-variables) or [CSS Shadow Parts](https://drafts.csswg.org/css-shadow-parts/).

I know it's more 1 to 1 with how Media Chrome does it but the controlsList API seems like a better fit without needing to introduce another attribute and property.

Reply:
I'm down for that idea. I'll go ahead and refactor, but just want to mention as food for thought that this will be the one controlslist element without a corresponding part. (not a big deal in my opinion)